docs/benchmarks/data-serving.md

```diff
@@ -41,6 +41,8 @@ $ docker run --name cassandra-server(id) --net serving_network -e CASSANDRA_SEED
 
 You can find more details at the websites: http://wiki.apache.org/cassandra/GettingStarted and https://hub.docker.com/_/cassandra/.
 
+Make sure all non seed servers are stablished - adding them concurrently may lead to problem.
+
```

Comment:
Moving the polling to the container having YCSB installed led me to find it. In the following scenario the second non-seed machine can't join the cassandra network: 

1. Start a seed server, 
2. start a non-seed server using the seed server,
3. immediately start another similar non seed server.

Others had the same experience and since it is up to the administrator and these changes does not happen frequently I just left a comment here. [Cassandra docs](https://docs.datastax.com/en/cassandra/2.1/cassandra/operations/ops_add_node_to_cluster_t.html).